### ISSUE TITLE
[4.0][a11y] Progress bar in Sample data module is not accessible

### ISSUE THREAD

**Issue by zwiastunsw** [joomla/joomla-cms#20996]
Screen reader does not announce progress information.
See:  
* Success Criteria [4.1.2 Name, Role, Value Level A](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=111#name-role-value)
* [ARIA4](https://www.w3.org/TR/WCAG20-TECHS/ARIA4.html): Using a WAI-ARIA role to expose the role of a user interface component
* [ARIA5](https://www.w3.org/TR/WCAG20-TECHS/ARIA5.html): Using WAI-ARIA state and property attributes to expose the state of a user interface component

**Comment by brianteeman** [comment]
**Sorry but this is of no use at all. Just sending a link to a generic page in the documentation is absolutely zero help at all.** 

What we need is specific information about what is not accessible and suggestions on how to make it accessible.

An example of that might be suggesting to use the progressbar role with a link to https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_progressbar_role 
or even better a link to the bootstrap documentation for this exact issue
https://getbootstrap.com/docs/4.0/components/progress/

**Comment by brianteeman** [comment]
Closed please test #21865
